test/Microsoft.Extensions.ML.Tests/FileLoaderTests.cs

```diff
@@ -10,13 +10,14 @@
 using Microsoft.Extensions.Options;
 using Microsoft.Extensions.Primitives;
 using Microsoft.ML.Data;
+using Microsoft.ML.TestFrameworkCommon.Attributes;
 using Xunit;
 
 namespace Microsoft.Extensions.ML
 {
     public class FileLoaderTests
     {
-        [Fact]
+        [MLNETFact]
```

Comment:
IMO this should only be applied to tests that are known to be flaky.

Reply:
+💯

Reply:
I agree. And I think that was the second proposal. To apply this only to tests known to be flaky and if we still see failures, to retry the whole configuration. 
Can you please amend the pull request as per your second proposal?

Reply:
I agree. Actually in the dotnet/runtime repo we kept it simple instead of adding a `FactAttribute` a `RetryHelper` class which forces people to use that when a test is flaky and it has to be justified. 

Also, that way you can also use it for theory tests not only facts.

https://github.com/dotnet/runtime/blob/master/src/libraries/Common/tests/CoreFx.Private.TestUtilities/System/RetryHelper.cs#L11

Reply:
Also, if we stick with an attribute I believe the name should express what the attribute does.

For example: `RetryFactAttribute` or `RetryAbleFactAttribute`.